hack/util.sh

```diff
@@ -458,10 +458,14 @@ function install_registry() {
 }
 readonly -f install_registry
 
-function wait_for_registry() {
+function wait_generation() {
```

Comment:
wait_for_synced_registry is a more appropriate name since you don't wait only for generation